README.md

```diff
@@ -61,6 +61,59 @@ If you have a preferred import path to suggest, append the suggestion after a `=
 -paths "fmt.{Errorf}=github.com/pkg/errors.{Errorf}"
 ```
 
+### Ignoring problems
+
+If you want to ignore a problem reported by `faillint` you can add a linter directive.
+The format of a directive is based on the format other linters, such as `staticcheck`, use and there are two types: line-based and file-based.
```

Comment:
I think this line is not applicable anymore and can be removed.

Reply:
Removed now, in 1c0756d783fa62720a308b12b9262cc9e310eeb3.